static/js/hotkey.js

```diff
@@ -515,6 +515,12 @@ exports.process_hotkey = function (e, hotkey) {
         }
     }
 
+    // Prevents up and down navigation along the message list when lightbox is open.
+    if ((event_name === 'up_arrow' || event_name === 'down_arrow') && overlays.lightbox_open()) {
+        e.preventDefault();
```

Comment:
But I think we  need the `perventDefault` when check if the `overlays.is_active` because by just return false we can still navigate upwards. In order to prevent that we have to include the `perventDefault`.